### ISSUE TITLE
Cannot choose character to play as in casual matches

### ISSUE THREAD

**Issue by dribbleondo** [rbjaxter/budhud#214]
[Can't select any character other than Heavy](https://steamuserimages-a.akamaihd.net/ugc/914668497182089546/E8122AB8D212A027F9D7E29382081CC454EE9724/)

This is a very odd bug.

**Comment by rbjaxter** [comment]
what in the world...

are you free to play? maybe this is a new f2p tutorial thing they're doing

**Comment by dribbleondo** [comment]
Nope, It just...sort of happened. No point looking into it now, as todays update fixed the issue.

**Comment by rbjaxter** [comment]
oh, glad it resolved itself!

**Comment by dribbleondo** [comment]
[It happened again. What on earth is causing this!?](http://steamcommunity.com/sharedfiles/filedetails/?id=1347678956)

**Comment by rbjaxter** [comment]
That dialog appears to be associated with the training mode. I'm...not sure what could be causing it. Quite a strange issue :\. I'll try to look into it a bit.

**Comment by dribbleondo** [comment]
As a bit of an update, removing the #215 files entirely and reinstalling budhgud with #216 made Budhud work again on the latest version. It seems to be a copy+paste over existing files problem.

**Comment by rbjaxter** [comment]
so this may not have been the issue you were experiencing, but there's a file titled `trainingprogress.txt` that lives in `\Team Fortress 2\tf\` that may have had some minor part in this bug. I could be wrong, though

**Comment by dribbleondo** [comment]
Well, we'll soon find out. Gonna download the latest dev version of budhud to see if the bug reappears
